features/rules/BBX/BBX000_Bounding-box.feature

```diff
@@ -0,0 +1,35 @@
+@implementer-agreement
+@BLT
```

Comment:
```suggestion
@GEM
```

I suppose?

Reply:
BBX, we need that for scorecards

Reply:
Doesn't it have to be both BBX and GEM?
This is the case in BRP, e.g. https://github.com/buildingSMART/ifc-gherkin-rules/blob/development/features/rules/BRP/BRP003_Planar-faces-are-planar.feature

Furthermore, I noticed that BRP002 only has a `@GEM` tag, so we should probably change this too. 
https://github.com/buildingSMART/ifc-gherkin-rules/blob/development/features/rules/BRP/BRP002_Single-component-in-connected-faceset.feature

Reply:
IIRC we decided to only go for one tag per rule. 
So the current tag should be `BBX` and BRP002 should be `@BRP` only